### DISCUSSION TITLE
dist directory, files and paths

### DISCUSSION THREAD

**Discussion by FranGomezVenegas** [open-wc/open-wc#2223] [Q&A]
Hi All,

I create one build and all the files are added to the dist directory, that's fine, index.html and the all js files in the same root.

Then I want to upload the dist directory to my tomcat file server and I open the index.html and it opens but then it does not find the js files because it looks for them in the root, tomcat webapps, not in the dist directory.

How could I do to let the build look at the js files into the dist directory?

I found the way to create the build in othre directory name than dist but anyway it happens the same because it is only to rename the main build directory name.

Any help would be great! Thanks in advance

**Comment by gjevers** [comment]
@FranGomezVenegas Did you solve this issue? I'm struggling with the same issue.

**Comment by Westbrook** [comment]
Don’t move the `dist` directory, move all of the thing IN that directory to the root.
